libraries/ui-library/src/components/six-button/six-button.tsx

```diff
@@ -135,7 +135,7 @@ export class SixButton {
     const Button = isLink ? 'a' : 'button';
 
     return (
-      <div class={{ 'button-wrapper--disabled': this.disabled }}>
+      <div onClick={this.handleClick} class={{ 'button-wrapper--disabled': this.disabled }}>
```

Comment:
would it be possible to add a unit test which catches this, i.e. makes sure no click is emitted for disabled button?

Reply:
Would be great, but I think we would need to add end-to-end tests for this, which we currently don't have.